test/suite/test_hs19.py

```diff
@@ -52,7 +78,7 @@ def test_hs19(self):
         # to reconstruct the final value, corrupting the resulting value.
         self.session.begin_transaction()
         cursor.set_key(str(0))
-        mods = [wiredtiger.Modify('AAAAAAAAAA', 102, 10)]
+        mods = [wiredtiger.Modify('AAAAAAAAAA', 102, 0)]
```

Comment:
Sorry - I changed this because I didn't understand it. Could you explain what this parameter does?

Reply:
This parameter specifies the number of bytes to be replaced by the modify. 0 results in the modify being inserted.